### ISSUE TITLE
Help icon text no longer displays - same issue in developer version

### ISSUE THREAD

**Issue by PercyP** [YetiForceCompany/YetiForceCRM#5080]
![testsave2](https://user-images.githubusercontent.com/10598886/29839398-f5acdb66-8cf5-11e7-86fd-2ee33e562db9.png)
Hi,

the Help Icon text in 

System-Language-Help Icon 

Does not display in the records. The text saves (see the 2 test help icons I did under Contacts - Salutation and Name
![testsave1](https://user-images.githubusercontent.com/10598886/29839395-f051dd7e-8cf5-11e7-8ed4-502cd3bfbaf4.png)

**Comment by rskrzypczak** [comment]
It's working for me
![image](https://user-images.githubusercontent.com/10221999/29841283-268d14da-8d05-11e7-9c96-6a22f45ad261.png)

**Comment by PercyP** [comment]
Hi,

did you test it in the git developer? It still doesn't work for me so that's odd. I tried with US English and Polish but nothing displays for me

**Comment by mariuszkrzaczkowski** [comment]
Record a video of what you are doing

**Comment by PercyP** [comment]
Hi @mariuszkrzaczkowski 
I have recorded a video using screencast and saved it as a gif file but it says its too big 15MB) so I have sent it to Paulina.

**Comment by PercyP** [comment]
Hi @mariuszkrzaczkowski 

the video got rejected by email too - too big. What are people using on here to create videos? 

All I did was select everything in Show in, then added some text in the Edit section, then closed it. Checked that the text appears by hovering over the icon, then went to Contacts, Last Name and it didn't appear. Am I missing a step? There was no Save button or anything extra I could see.

**Comment by mariuszkrzaczkowski** [comment]
We recommend ScreenToGif only set less frames per second and do not record long movies

**Comment by PercyP** [comment]
@mariuszkrzaczkowski 

I can see what the problem is now. It will not display the help icon text that is added to a field in the Basic details
![helic](https://user-images.githubusercontent.com/10598886/29959055-83f293fe-8eed-11e7-87d7-600f0cde7726.gif)
 block, but it will display it if it is added to a field in one of the other blocks. To check this I added help icon text to the Last Name, Job Title in Basic Details block and these don't display, then I added text to the Mobile field and that does display.
